ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -89,124 +106,111 @@ def greater_equal(x1, x2, /):
 
 
 @to_ivy_arrays_and_back
-def isnan(x, /):
-    return ivy.isnan(x)
+def invert(x, /):
+    return ivy.bitwise_invert(x)
 
 
 @to_ivy_arrays_and_back
-def equal(x1, x2, /):
-    x1, x2 = promote_jax_arrays(x1, x2)
-    return ivy.equal(x1, x2)
+def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
+    a, b = promote_jax_arrays(a, b)
+    return ivy.isclose(a, b, rtol=rtol, atol=atol, equal_nan=equal_nan)
 
 
 @to_ivy_arrays_and_back
-def all(a, axis=None, out=None, keepdims=False, *, where=False):
-    return ivy.all(a, axis=axis, keepdims=keepdims, out=out)
+def iscomplex(x: any):
+    return ivy.bitwise_invert(ivy.isreal(x))
 
 
 @to_ivy_arrays_and_back
-def bitwise_and(x1, x2, /):
-    x1, x2 = promote_jax_arrays(x1, x2)
-    return ivy.bitwise_and(x1, x2)
+def iscomplexobj(x):
+    return ivy.is_complex_dtype(ivy.dtype(x))
 
 
 @to_ivy_arrays_and_back
-def bitwise_not(x, /):
-    return ivy.bitwise_invert(x)
+def isfinite(x, /):
+    return ivy.isfinite(x)
 
 
 @to_ivy_arrays_and_back
-def bitwise_or(x1, x2, /):
-    x1, x2 = promote_jax_arrays(x1, x2)
-    return ivy.bitwise_or(x1, x2)
+def isin(element, test_elements, assume_unique=False, invert=False):
+    return ivy.isin(element, test_elements, assume_unique=assume_unique, invert=invert)
 
 
 @to_ivy_arrays_and_back
-def bitwise_xor(x1, x2, /):
-    x1, x2 = promote_jax_arrays(x1, x2)
-    return ivy.bitwise_xor(x1, x2)
+def isinf(x, /):
+    return ivy.isinf(x)
 
 
 @to_ivy_arrays_and_back
-def any(a, axis=None, out=None, keepdims=False, *, where=None):
-    # TODO: Out not supported
-    ret = ivy.any(a, axis=axis, keepdims=keepdims)
-    if ivy.is_array(where):
-        where = ivy.array(where, dtype=ivy.bool)
-        ret = ivy.where(where, ret, ivy.default(None, ivy.zeros_like(ret)))
-    return ret
-
-
-alltrue = all
-
-sometrue = any
-from ivy.functional.frontends.jax.numpy import promote_types_of_jax_inputs
```

Comment:
Not really sure about this import and why it's here, the formatter did move it to the beginning of the file. Thought I should flag that

Reply:
Well all imports should indeed be at the top, but this one is a duplicate. I will remove it. Thanks!